.github/workflows/main.yml

```diff
@@ -145,3 +145,32 @@ jobs:
                 VERSION: "$VERSION",
               }
             })
+
+
+  trigger_kolibri-installer-android:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/') || startsWith(inputs.tagname, 'v')
+    needs: [build_apps-bundle, build_kolibri-explore-plugin_whl]
+    env:
+      branch: ${{ github.ref }}
+
+    steps:
+      - name: Set branch if this is triggered by bump2release.yml
+        if: startsWith(inputs.tagname, 'v')
+        run: |
+          # The kolibri-explore-plugin release flow from bump2release.yml will
+          # append a new commit to master branch. Therefor, the github.ref is
+          # not at the latest commit. Checkout master instead.
+          echo "branch=${{ inputs.tagname }}" >> $GITHUB_ENV
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.branch }}
+
+      - name: Trigger building kolibri-installer-android
+        run: |
+          scripts/trigger_jenkins.py --build-url kolibri-installer-android UPLOAD=true
```

Comment:
I don't know how `PATH` is setup, but I suspect you want to run this as `./scripts/trigger_jenkins.py`.

Also, I doubt the runner is going to have all of `python3`, `requests` and `python-jenkins` installed, so running this script will just fall over. I think for such a simple case you could just add another step before this that installs the prereqs from Ubuntu:

```
run: |
  sudo apt update
  sudo apt install -y python3 python3-jenkins python3-requests
```